requirements/runtime.txt

```diff
@@ -5,6 +5,7 @@ mmcls>=0.16.0
 motmetrics
 packaging
 pandas<=1.3.5
+psutil
```

Comment:
Why need this package

Reply:
If it is caused by https://github.com/open-mmlab/mmtracking/blob/dev-1.x/mmtrack/utils/benchmark.py#L45, we can modify the typehint to `p: 'psutil.Process'`